compiler-cli/src/lsp.rs

```diff
@@ -715,6 +717,45 @@ impl LanguageServer {
     }
 }
 
+#[cfg(target_os = "windows")]
+fn uri_to_module_name(uri: &Url, root: &Path) -> Option<String> {
+    let mut uri_path = decode(&*uri.path().replace('/', "\\"))
```

Comment:
What does the decode bit do here?

Reply:
For some reason parts of the path are URL encoded, for example the colon is shown as %3A which causes issues.